dagr/dagr.py

```diff
@@ -155,9 +155,13 @@ def get(self, url, file_name=None):
             mod_time = mktime(parsedate(get_resp.headers.get("last-modified")))
             utime(file_name, (mod_time, mod_time))
 
-        if get_resp.headers.get("content-type"):
-            rename(file_name, file_name + guess_extension(
-                get_resp.headers.get("content-type").split(";")[0]))
+        contentType = get_resp.headers.get("content-type")
+        if contentType:
+            file_ext = guess_extension(contentType.split(";")[0])
+            if (file_ext is None):
+                file_ext = "." + contentType.split("/")[1]
```

Comment:
This will not work if content type is not only media type (which is the reason for the split on ";"). Something like that should cover all cases:
`if content_type:
    media_type = content_type.split(";")[0]
    file_ext = guess_extension(media_type)
    if not file_ext :
        file_ext = "." + media_type.split("/")[1]`